src/components/Toast/Toast.tsx

```diff
@@ -91,6 +92,11 @@ const Toast: React.FC<ToastProps> = ({ onClose, children }) => (
 
 Toast.propTypes = {
   onClose: PropTypes.func.isRequired,
+  width: PropTypes.number,
+};
+
+Toast.defaultProps = {
```

Comment:
Please drop the `.defaultProps` and set defaults directly in the component, we are using default props only when we export directly styled component.
```js
const Toast: React.FC<ToastProps> = ({ onClose, children, width = 370 }) =>